examples/graphiql-create-react-app/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "example-graphiql-create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/react": "^16.9.34",
+    "@types/react-dom": "^16.9.6",
+    "graphiql": "file:../../packages/graphiql",
```

Comment:
this should be `file:packages/graphiql` now, my bad

Reply:
to confirm this resolves correctly in yarn workspaces, run `yarn workspaces info` and make sure there aren't any `mismatchedDependencies`

Reply:
> this should be `file:packages/graphiql` now, my bad

That doesn't seem to work. 
It throws a error `Package "graphiql" refers to a non-existing file . . .`. 
It points to the `packages/graphiql` inside the `examples/cra`

Reply:
There aren't any `mismatchedDepedencies`

Reply:
ok, I think I had something backwards then, my bad, carry on!